docs/docs/components/application-wrapper.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
This can be easily solved by typescript I guess instead of using this `prop-types` dependency.
But I'm not sure I understand what all this react code is... Can you help me understand?